### DISCUSSION TITLE
per le spese come faccio a individuare quelle scaricate da SDI ? c'è un endpoint con una risposta che le individua ?

### DISCUSSION THREAD

**Discussion by crismas71** [fattureincloud/api#478] [Q&A]
Relativamente ai received_documents c'è una informazione che mi permette di distinguere le fatture scaricate da SDI rispetto a quelle ad esempio che sono state caricate manualmente ?
grazie

**Comment by valmoz** [comment]
Buongiorno @crismas71 
mi spiace, questa informazione non è attualmente disponibile.

Grazie
Mauro

**Reply by crismas71** [reply]
ok grazie. l'endpoint received_documents espone gli importi solo se la fattura viene registrata giusto ? se si vi è un campo che indica che la fattura è stata registrata ? grazie
